migrations/00011_add_refresh_token_hash_column.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+ALTER TABLE "auth"."refresh_tokens"
+    ADD COLUMN "hashed_refresh_token" VARCHAR(255) GENERATED ALWAYS AS (sha256 (refresh_token::text::bytea)) STORED;
```

Comment:
1. Why do we generate this value instead of storing the hash? Storing the hash seems more sensible because that's what we'll do in the next major release.
2. I think the column should be named `refresh_token_hash` to be consistent with the naming of `password_hash` (https://github.com/nhost/hasura-auth/blob/main/migrations/00001_create-initial-tables.sql#L59).

Reply:
1. Postgres generated columns are actually stored into the database. To "convert" them into standard columns is straight-forward as we only need to remove the generated expression. This approach reduces the burden of having to keep the two columns in sync in hasura-auth during the transition period. We will have to create a new migration regardless, as we will change the primary key name from `refresh_token` to `id`.
2. makes sense, will change this

Reply:
1. sounds good
2. check